AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-table/vho-work-hours-table.component.ts

```diff
@@ -161,4 +170,14 @@ export class VhoWorkHoursTableComponent implements CanDeactiveComponent {
     get checkVhoHasWorkHours(): boolean {
         return this.workHours?.length > 0;
     }
+    showMessage(message: string) {
```

Comment:
these methods have been introduced but I do not see any new tests to accompany them

Reply:
a unit test is covering this .  it('should display a message', () => {